{{ cookiecutter.package_name }}/docs/conf.py

```diff
@@ -6,12 +6,23 @@
 
 import datetime
 
+from packaging.version import Version
+
 # -- Project information -----------------------------------------------------
 
 # The full version, including alpha/beta/rc tags
 from {{ cookiecutter.module_name }} import __version__
 
-release = __version__
+_version_ = Version(__version__)
+# NOTE: Avoid "post" appearing in version string in rendered docs
+if _version_.is_postrelease:
+    version = release = f'{_version_.major}.{_version_.minor}.{_version_.micro}'
```

Comment:
I think we'd be better off using the `base_version` property and then appending dev back on: https://packaging.pypa.io/en/stable/version.html#packaging.version.Version.base_version

Reply:
Oh I completely missed that property. I agree.